content/en/docs/concepts/security/index.md

```diff
@@ -162,7 +162,7 @@ Istio provides two types of authentication:
     - [Google Auth](https://developers.google.com/identity/protocols/OpenIDConnect)
 
 In all cases, Istio stores the authentication policies in the `Istio config
-store` via a custom Kubernetes API. Istiod keeps them up-to-date for each proxy,
+store` via a custom Kubernetes API. [Istiod](/docs/reference/glossary/#istiod) keeps them up-to-date for each proxy,
```

Comment:
Instead of links we reference glossary entries using a hover mechanism.
```suggestion
store` via a custom Kubernetes API. {{< gloss >}}Istiod{{< /gloss >}} keeps them up-to-date for each proxy,
```

Reply:
Ah, sorry.. I'll adjust.

Reply:
updated with https://github.com/istio/istio.io/pull/6899/commits/801b7a0e4665ec60b22a9f6e63a7b3227b588da0